### PR TITLE
adding footer

### PR DESCRIPTION
**Pull request by Vinz2406**
Creation of the footer with addition of Footer.jsx and Footer.css files

### CODE REVIEW

File: client/src/components/Footer.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.social_networks {
+  display: flex;
+  gap: 10px;
+  font-size: 30px;
+  color: #07b2d9;
+}
+
+.social_networks .fa-icon:hover {
```
Comment (alexandreLebeau) [inline client/src/components/Footer.css]: replace .fa-icon with .fa_icon

**Review by alexandreLebeau — CHANGES_REQUESTED** [review]

File: client/src/App.jsx [hunk 2]
```diff
@@ -17,7 +18,13 @@ function App() {
       });
   }, []);
 
-  return <div>{beersList && <CardsList beerList={beersList} />}</div>;
+  return (
+    <div>
+      {beersList && <CardsList beerList={beersList} />}
+
```
Comment (fantinerudent) [inline client/src/App.jsx]: useless empty line

**Review by fantinerudent — APPROVED** [review]

**Comment by alexandreLebeau** [comment]
all good

**Review by alexandreLebeau — APPROVED** [review]
all good

**Pull request merged**